android/engine/src/main/java/org/smartregister/fhircore/engine/di/FhirTaskExpireEntryPoint.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.di
+
+import com.google.android.fhir.FhirEngine
+import dagger.hilt.EntryPoint
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import org.smartregister.fhircore.engine.task.FhirTaskExpireUtil
+
+/** Created by Ephraim Kigamba - nek.eam@gmail.com on 05-12-2022. */
+@InstallIn(SingletonComponent::class)
+@EntryPoint
+interface FhirTaskExpireEntryPoint {
+
+  fun fhirEngine(): FhirEngine
+
+  fun fhirTaskExpireUtil(): FhirTaskExpireUtil
+}
```

Comment:
Was this required for dependency injection? Dependencies can be injected to a subclass  of `CoroutineWorker` as constructor parameters.

Reply:
I believe so but I spent a lot of time trying it but couldn't get it to work. 

You can possibly help to get this working. The current injection works almost similar and depends on Hilt